test/runtime.js

```diff
@@ -173,22 +206,44 @@ describe('runtime parameters', function() {
         assert.equal(process.sass.binaryUrl.substr(0, URL.length), URL);
       });
     });
-    describe('checking if environment variable overrides package.json', function() {
+    describe('checking if environment variable overrides .npmrc configuration variable', function() {
```

Comment:
I don't think either approach is correct. We no have 4 ways to set these configurations. Testing they get set is important, testing their precedence is most important. The better tests would be asserting, per variable, that precedence is respected.

```
describe('configuration precedence should be respected', () => {
  describe('SASS_BINARY_PATH', () => {
    beforeEach(() => {
      process.env.SASS_BINARY_PATH = 'foo';
      process.env.npm_config_sass_binary_path = 'bar';
      require.cache[packagePath].exports.nodeSassConfig = { binaryPath: 'baz' };
      require(extensionsPath);
    });
    it('environment variable', () => {
      assert.equal(process.sass.binaryPath, 'baz');
    });
    it('npm config', () => {
      process.env.npm_config_sass_binary_path = null;
      assert.equal(process.sass.binaryPath, 'bar');
    });
    it('package.json', () => {
      process.env.npm_config_sass_binary_path = null;
      require.cache[packagePath].exports.nodeSassConfig = {};
      assert.equal(process.sass.binaryPath, 'foo');
    });
  })
})
```